README.md

```diff
@@ -120,18 +121,23 @@ Enable the service account to [access the Traffic Director API](https://cloud.go
 ```shell
 gcloud projects add-iam-policy-binding "${PROJECT_ID}" \
    --member="serviceAccount:${WORKLOAD_SA_EMAIL}" \
-   --role="roles/trafficdirector.client"
+   --role="roles/trafficdirector.client" \
+   --condition="None"
```

Comment:
Just curious, why do we need this?

Reply:
Haha, good question. Because we have [a condition on one of iam bindings](https://github.com/grpc/psm-interop/blob/main/README.md#allow-test-driver-to-configure-workload-identity-automatically) in the project (!), gcloud tool always goes into the interactive mode and asks you if you want to select that condition, or None. Very annoying IMO.